### DISCUSSION TITLE
It is strange. The image saved as png through pillow or cv2 is very slow to load.                    pix = page.get_pixmap(clip=bbox, dpi=300)  # pixmap of line bbox

### DISCUSSION THREAD

**Discussion by nissansz** [pymupdf/PyMuPDF#2659] [Looking for help]
It is strange. The image saved as png through pillow or cv2 is very slow to load.                    pix = page.get_pixmap(clip=bbox, dpi=300)  # pixmap of line bbox

**Comment by JorjMcKie** [comment]
How about adding more flesh to this comment 😎?

**Comment by nissansz** [comment]
I just don't know why,
image from video is quick to load in paddle training.
Image from mupdf is slow to load in paddle training.

**Reply by JorjMcKie** [reply]
This post is not very useful currently.
Please add more information if you want help on this.
For example, add the two image files where you observed that drastically different behaviour.

**Comment by nissansz** [comment]
I will use jpg for this case. Though in some other cases, png is much smaller than jpg.
